src/workspaces/cookiefactory/sample_dashboards/mixer_alarms_dashboard.json

```diff
@@ -21,65 +21,30 @@
   "description": "",
   "editable": true,
   "fiscalYearStartMonth": 0,
-  "gnetId": null,
   "graphTooltip": 0,
   "id": 1,
-  "iteration": 1637731351082,
+  "iteration": 1650938759667,
   "links": [],
   "liveNow": false,
   "panels": [
     {
       "collapsed": false,
-      "datasource": null,
       "gridPos": {
         "h": 1,
         "w": 24,
         "x": 0,
         "y": 0
       },
-      "id": 49,
-      "panels": [],
-      "title": "Introduction",
```

Comment:
Looks like this section was removed? Was actually wondering if we wanted to expand it to briefly introduce the new feature being added to the dashboard?